### ISSUE TITLE
Follow Up on performance note

### ISSUE THREAD

**Issue by randeepbydesign** [MarcoGhise/ReactiveMongoPerformance#1]
Hi Marco:

I wanted to thank you for this project and to follow up to hear if you heard anything from the Reactive community on your test?

**Comment by MarcoGhise** [comment]
Hi.

Thank you for your comment. Have a look at the article which illustrates the repository code at
https://techannotation.wordpress.com/2018/05/14/spring-reactive-performance-benchmark/

At the UPDATE section, I summarize the chat that I had with Moisés Macero. I hope you get what you're looking for!

**Comment by liucyu** [comment]
i agree with you  because we also encountered this problem，and performance lost 100 ~500ms，but the system load dropped 4 times
